source/atlas-cli-deploy-docker.txt

```diff
@@ -432,3 +439,55 @@ directory:
             sudo dpkg -i mongosh.deb
             mongosh --version
         - run: mongosh 'mongodb://localhost/?directConnection=true' --eval 'show dbs'
+
+
+.. _convert-to-local-dev:
+
+Convert Official Image to an Atlas Local Dev Implementation
```

Comment:
I think the feature is being called "Local Atlas Deployment" in https://www.mongodb.com/docs/atlas/cli/current/atlas-cli-deploy-local/
```suggestion
Convert Official Image to a Local Atlas Deployment
```